frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -186,36 +182,36 @@ class PopupCreateFile extends React.Component {
 
     xhr.onreadystatechange = () => {
       if (xhr.readyState === 4) {
-        const uploadedFiles = state.uploadedFiles
+        const uploadedFilesUpdated = state.uploadedFiles
```

Comment:
above in the code, you used "List" to tell that this variable is an array. Try and keep consistency in your naming